packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -352,6 +364,16 @@ const CreatePassword = ({ password, platform, validatePassword, onSubmit, error_
                                 )}
                             </PasswordMeter>
                         </div>
+                        {props.is_real_financial_stp && !props.is_bvi && (
```

Comment:
Here we need to check if  the client has Financial or Synthetic account, then check is_bvi

Reply:
Done. Please check